app/_services/dates.ts

```diff
@@ -1,9 +1,11 @@
 import dayjs from 'dayjs';
 import bigIntSupport from 'dayjs/plugin/bigIntSupport';
 import localizedFormat from 'dayjs/plugin/localizedFormat';
+
 dayjs.extend(bigIntSupport);
 dayjs.extend(localizedFormat);
 
 export function formatTimestampSecondsAsLocal(timestampSeconds: bigint) {
-	return dayjs(timestampSeconds * BigInt('1000')).format('L LT');
+	const date = dayjs(timestampSeconds * BigInt('1000'));
```

Comment:
what is dayjs for?

why can't you just do `new Date(..)`?

Reply:
Not my implementation, but it can be removed, yes